conflowgen/posthoc_analyses/inbound_and_outbound_vehicle_capacity_analysis_report.py

```diff
@@ -53,11 +58,10 @@ def get_report_as_graph(self) -> object:
              The matplotlib axis of the bar chart.
         """
 
-        import pandas as pd  # pylint: disable=import-outside-toplevel
-        import seaborn as sns  # pylint: disable=import-outside-toplevel
-        sns.set_palette(sns.color_palette())
-
         inbound_capacities, outbound_actual_capacities, outbound_maximum_capacities = self._get_capacities()
+        if (len(inbound_capacities) + len(outbound_actual_capacities) + len(outbound_maximum_capacities)) == 0:
```

Comment:
Again, why did you choose `len`? Each `capacities` object is actually a dictionairy. We initially introduced this check because we wanted to prevent a division by zero. Please check the report whether we even need this specific graph. Whenever a bar chart is used, having several bars at 0 is no problem so we do not need the hint that no data is available.